packages/orbit-components/config/build/consts.mts

```diff
@@ -14,7 +14,8 @@ export const COMPILE_IGNORE_PATTERNS = [
   // TODO: remove that after TW migration, tailwind is temporary folder
   "**/tmp-tailwind/**/*.{ts?(x),d.ts}",
   "**/*.d.ts",
-  "**/*.stories.*",
```

Comment:
this shouldn't be removed, we still have storybook and stories shouldn't be in the builded output 🙂

Reply:
ah lol don't know why it was removed